package.json

```diff
@@ -13,7 +13,8 @@
     "test": "vitest",
     "coverage": "vitest run --coverage",
     "prepublishOnly": "pnpm run clean && pnpm install && pnpm run check",
-    "generate:rules": "tsx ./scripts/generate-rule-files/cli.ts"
+    "generate:rules": "tsx ./scripts/generate-rule-files/cli.ts",
+    "regenerate:rules": "del-cli ./src/rules/*/ && pnpm run generate:rules && pnpm run format && pnpm run lint && pnpm run typecheck --run"
```

Comment:
If we add something like this, I would like to use the same strategy as in another project of mine:

https://github.com/faker-js/faker/blob/cddbb959f13aa29a7264174b1fc4eaa4271cf900/package.json#L85

There we have this `preflight` command that manages to run every important script

Beside this preflight we might then need a `generate:clean` 🤔 

I also don't like to use `del-cli` as a new dependency but just `rm -Rf` (or introduce `rimraf` if really necessary)

Maybe we should do all these in a separate PR because I still do squash-merge PRs

Reply:
Well, I'm on Windows. Thus I need something. And that one path delimiter fix.